hamocc/mo_inventory_bgc.F90

```diff
@@ -481,10 +481,10 @@ subroutine inventory_bgc(kpie,kpje,kpke,dlxp,dlyp,ddpo,omask,iogrp)
              &                   - srivflux(iralk) ! no DIN & DIP substraction because alkalinity
                                                    ! changes due to instantaneous remineralisation 
                                                    ! of riverine DOC are ignored
-        totalnitr   = totalnitr  - srivflux(irdoc)*rnit-srivflux(irtdoc)*rnit_tdochc               &
-             &                   - srivflux(irdet)*rnit_tdoclc - srivflux(irdin)
-        totalphos   = totalphos  - srivflux(irdoc)-srivflux(irtdoc)-srivflux(irdet)-srivflux(irdip)
-        totaloxy    = totaloxy   - srivflux(irdoc)*(-24.) - srivflux(irtdoc)*(-49.5)               &
+        totalnitr   = totalnitr  - srivflux(irtdoc)*rnit_tdochc-srivflux(irdet)*rnit_tdoclc        &
+             &                   - srivflux(irdin)
+        totalphos   = totalphos  - srivflux(irtdoc)-srivflux(irdet)-srivflux(irdip)
+        totaloxy    = totaloxy   - (srivflux(irdoc)+srivflux(irtdoc))*(-49.5)                      &
```

Comment:
Isn't the oxygen for the remineralization for DOC beforehand consumed? - I might still not fully get the protocol, but since the irdoc is beforehand put into the DIN and DIP, why is the oxygen contained here considered?